tests/xgboost/test_xgboost_autolog.py

```diff
@@ -249,49 +249,6 @@ def test_xgb_autolog_logs_metrics_with_early_stopping(bst_params, dtrain):
             assert metric_history == evals_result[eval_name][metric_name] + [best_metrics]
 
 
-@pytest.mark.large
-def test_xgb_autolog_batch_metrics_logger_logs_expected_metrics(bst_params, dtrain):
```

Comment:
This test case is unnecessary - we have several other test cases that ensure correct metrics are logged as a result of training. We now have queued / batched logging for metrics, params, & tags in addition to the preexisting `BatchMetricsLogger` for per-epoch metric logging from callbacks; having a test case asserting that a particular framework is used for particular metric values is not particularly valuable.